gtsfm/evaluation/metrics_report.py

```diff
@@ -62,9 +86,11 @@ def create_plots_for_distributions(metrics_dict: Dict[str, Any]) -> str:
     # Separate all the 1D distribution metrics.
     for metric, value in metrics_dict.items():
         if isinstance(value, dict):
-            distribution_metrics.append(metric)
+            all_nan_summary = all(v != v for v in value[metrics.SUMMARY_KEY].values())
```

Comment:
is there a better (more readable) way to check for nan?

Reply:
np.isnan() is more readable, but requires an import.